pnpm-lock.yaml

```diff
@@ -230,11 +230,11 @@ importers:
       '@mui/material': 5.10.14_thp4wrxo45nyllusrnt46mid4u
       '@mui/system': 5.10.14_v64nlmo6pkqfmpmxge6otzlg54
       '@mui/x-date-pickers': 5.0.8_p6r6qp7wldbu5ojdsmv4yukx2q
-      '@nhost/core': link:../packages/core
-      '@nhost/nextjs': link:../packages/nextjs
-      '@nhost/nhost-js': link:../packages/nhost-js
-      '@nhost/react': link:../packages/react
-      '@nhost/react-apollo': link:../packages/react-apollo
+      '@nhost/core': 0.9.4
```

Comment:
hmm, it's weird that it changes the links to actual versions here 🤔

Reply:
That's not clear, but I think this is related to the fact that `@nhost/` dependencies in the dashboard's `package.json` file are pointing to a specific version instead of pointing to `workspace:*`.
For some reason, our pnpm interpreted these versions as links in the lockfile (probably because it checked the local version first and saw it was meeting the dashboard's requirements) whereas Renovate pointed to the npm repository.

Reply:
An easy find to "solve" the issue would be to set the versions of the dashboard's internal dependencies to `workspace:*`, unless there is a valid reason not to

Reply:
By default, pnpm uses the workspace versions even if you don't set it to `workspace:*`, so the idea was to keep it as is, because pnpm takes care of everything. I'll change it to `workspace:*` now.